### PR TITLE
Add aos servicemanager recipe

### PR DESCRIPTION
**Pull request by al1img**

### CODE REVIEW

File: recipes-domx/meta-xt-images-domx/recipes-extended/aosservicemanager/github.com-cavaliercoder-grab_git.bb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+DESCRIPTION = "github.com/cavaliercoder/grab"
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-domx/recipes-extended/aosservicemanager/github.com-cavaliercoder-grab_git.bb:1]: nice description

File: recipes-domx/meta-xt-images-domx/recipes-extended/aosservicemanager/github.com-coreos-go-systemd_git.bb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+DESCRIPTION = "github.com/coreos/go-systemd"
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-domx/recipes-extended/aosservicemanager/github.com-coreos-go-systemd_git.bb:1]: ditto

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`
although I would love to see new recipes descriptions

**Comment by aanisov** [comment]
[s/dependancy/dependency/](https://github.com/xen-troops/meta-xt-images/pull/90/commits/0c486ac3520e4c5c7a4455e88cbfc6ed23fed202)

**Comment by aanisov** [comment]
BTW, I'm not really sure using ${AUTOREV} for outer repositories is a good idea. We are using it for our development repositories. But external components are taken with some specific version in order to get known functional version all the time.

**Pull request closed without merging**